src/std.rs

```diff
@@ -54,6 +54,14 @@ impl<T> List<T> {
             Err(_) => Err("<locked>"),
         }
     }
+
+    // Get the listener count by blocking
+    pub(crate) fn total_listeners_wait(&self) -> Result<usize, String> {
+        match self.0.lock() {
```

Comment:
I'm not sure if we want to lock an introduce contention here. Maybe we should use `try_lock` instead and return `0` if there is contention. Also, we shouldn't return `String` as that is a poor error type.

Reply:
If we return 0, and the mutex is busy, and in matter fact there are listeners listening. The user then used a false number and executed logic based on that.

Reply:
Alright, then that should be mentioned in the documentation.